plenum/server/node.py

```diff
@@ -4028,5 +3792,6 @@ def is_request_need_quorum(self, msg_dict: dict):
                                  self.is_action(txn_type) or
                                  self.is_query(txn_type))
 
-    def _init_write_request_validator(self):
-        pass
+    def _bootstrap_node(self, storage):
+        NodeBootstrap.__init__(self)
```

Comment:
Do we really need to use inheritance here for `NodeBootstrap`?

Reply:
We need to extend Node's functionality by extending "self" instance. Also, some of NodeBootstrap methods will be reloaded in indy-node's class.